website/docs/r/oraclepaas_database_service_instance.html.markdown

```diff
@@ -61,7 +61,7 @@ The following arguments are supported:
 * `default_access_rules` - (Optional) Specifies the details on which default access rules are enable or disabled. Default Access Rules
 are configured below.
 
-* `desired_state` - (Optional) Specifies the desired state of the servie instance. Allowed values are `start`, `stop`,
+* `desired_state` - (Optional) Specifies the desired state of the service instance. Allowed values are `start`, `stop`,
```

Comment:
should change to `running` and `shutdown`

However, I just noticed we added the DB running state in v1.1.0, so this is already out there. Could we "deprecate" the use of start/stop (with a warning) and add running/shutdown as the preferred values.